umh-core/pkg/service/httpclient/httpclient.go

```diff
@@ -28,6 +28,29 @@ import (
 	"go.uber.org/zap"
 )
 
+var (
+	// defaultTransport is a shared transport with connection pooling
+	defaultTransport = &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
+		DialContext: (&net.Dialer{
+			Timeout:   500 * time.Millisecond,
```

Comment:
why did you selected the timeouts here? why 500 milliseconds when the ticker is 100 ms

Reply:
It could be longer here, as the conn pool will not close, if a single conn is aborted, leaving the tcp conn open for reuse.